app/grandchallenge/codebuild/templates/codebuild/buildspec.yml

```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2 &
+      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+  pre_build:
+    commands:
+      - docker build -t {{ algorithm_name }} {{ folder_name }}
```

Comment:
We should think about the repo and labels we use. `algorithm_name` might not be docker safe. I want to migrate from storing the images on S3 to private ECR, so will look up the format to use for the repo and see if we can pass this in. For the label we can generate a new UUID before building, and then use this UUID when the `AlgorithmImage` is created.

Reply:
`algorithm_name` should be docker save as it's lower case and all non alphanumerical characters are replaced by dashes. It really does not matter much what we use here though, so a UUID would work as well.